tests/integration/test_rpm.py

```diff
@@ -105,6 +106,71 @@ def test_rpm_packages(
     )
 
 
+@pytest.mark.parametrize(
+    "test_params",
+    [
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachi2-rpm",
+                ref="4006ca755eee5bab541814b496121ff52598c818",
+                packages=({"path": ".", "type": "rpm"},),
+                flags=["--dev-package-managers"],
+                check_output=False,
+                check_deps_checksums=False,
+                check_vendor_checksums=False,
+                expected_exit_code=0,
+            ),
+            id="rpm_test_repo_file",
+        ),
+    ],
+)
+def test_repo_files(
+    test_params: utils.TestParameters,
+    cachi2_image: utils.ContainerImage,
+    tmp_path: Path,
+    test_data_dir: Path,
+    request: pytest.FixtureRequest,
+) -> None:
+    """Test if the contents of the generated .repo file are correct."""
+    test_case = request.node.callspec.id
+    output_folder = tmp_path.joinpath(f"{test_case}-output")
+
+    source_folder = utils.clone_repository(
+        test_params.repo, test_params.ref, f"{test_case}-source", tmp_path
+    )
+
+    utils.fetch_deps_and_check_output(
+        tmp_path, test_case, test_params, source_folder, test_data_dir, cachi2_image
+    )
+
+    # call inject-files to create the .repo file
+    cmd = [
+        "inject-files",
+        output_folder,
+        "--for-output-dir",
+        Path("/tmp").joinpath(f"{test_case}-output"),
```

Comment:
can't `tmp_path` be used here as well?

Reply:
It can, but then we'd need to deal with its randomness. FWIW, we are not using this path at all, it is just a means of having consistent data for the test (also, I just copied this code from the [utils](https://github.com/containerbuildsystem/cachi2/blob/main/tests/integration/utils.py#L394-L399) module).

Reply:
Doh! I missed the obvious connection to the expected .repo file you're creating.

Still, a nitpick - we could just write this as `Path("/tmp", f"{test_case}-output")` as `joinpath` makes everything visually longer and with `Path` and `/` I guess it only exists for completeness :), you don't need to change anything though, just my 2c.